www/src/pages/components/text-area/android/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Text Area
+description: Multiline inputs for text.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+`ThumbprintTextArea` is a multi-line ThumbprintEditText. For iOS and web parity, this is referred to as a Text Area. Since `ThumbprintTextArea` inherits from `ThumbprintEditTextBase`, it shares common properties with `ThumbprintTextInput`.
+
+### States
+
+The following states are available for inputs: Default, selected, filled, disabled, and error.
+
+![Text Area states](/img/native-text-input-states.png 'text-input states')
```

Comment:
This image shows the single line text-input component. Do you need to update it for TextArea?

Reply:
Ahh I was looking at the wrong part of the Figma file. Done and added.